react/features/gifs/function.any.ts

```diff
@@ -54,33 +45,50 @@ export function getGifForParticipant(state: IReduxState, participantId: string):
     return isGifEnabled(state) ? state['features/gifs'].gifList.get(participantId) || {} : {};
 }
 
+/**
+ * Returns true if a given URL is allowed to be rendered as gif and false otherwise.
+ *
+ * @param {string} url - The URL to be validated.
+ * @returns {boolean} - True if a given URL is allowed to be rendered as gif and false otherwise.
+ */
+export function isGifUrlAllowed(url: string) {
+    let hostname: string | undefined;
+
+    try {
+        const urlObject = new URL(url);
+
+        hostname = urlObject?.hostname;
+    } catch (_error) {
+        return false;
+    }
+
+    return Boolean(hostname) && Boolean(GIF_HOST_WHITELIST.find(h => h === hostname));
```

Comment:
Since it's oonly 1 domain, it's more resilient to do an equaality check. This way any prototype pollution bug doesn't render this protection useless.

Reply:
OK